---
widget: blank
widget_id: Softwares
headless: true
weight: 50
title: Softwares
active: true
design:
  columns: "2"
  background:
    text_color_light: false
    image_darken: 0
---

**Software:**

[HH-suite](https://github.com/soedinglab/hh-suite "hhsuite GitHub"): A sensitive protein sequence searching algorithm using hidden Markov models. [(read more here)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3019-7)

[MMseqs2](https://mmseqs.com/ "mmseqs2"): A ultra fast and sensitive sequence searching and clustering method. [(read more here)](https://academic.oup.com/bioinformatics/article/32/9/1323/1744460)

[Plass](https://github.com/soedinglab/plass): _De novo_ protein-level assembler for metagenomic samples. [(read more here)](https://www.nature.com/articles/s41592-019-0437-4)

[CCMpred](https://github.com/soedinglab/CCMpred "compared"): A fast and high precision tool to find protein residue-residue Contacts based on correlated Mutations. [(read more here)](https://academic.oup.com/bioinformatics/article/30/21/3128/2422261)

[WIsH](https://github.com/soedinglab/wish): A prediction tool for host-phage relationships. [(read more here)](https://academic.oup.com/bioinformatics/article/33/19/3113/3964377)

[kClust](https://github.com/soedinglab/kClust): A fast and sensitive clustering method for protein sequences. [(read more here)](https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-248)

[spacepharer](https://github.com/soedinglab/spacepharer): A sensitive tool for identifying host-phage pairs based on CRISPR spacers. [(read more here)](https://academic.oup.com/bioinformatics/article/37/19/3364/6207963)

[MetaEuk](https://github.com/soedinglab/metaeuk): A sensitive, high-throughput gene discovery, and annotation tool for large-scale eukaryotic metagenomics. [(read mode here)](https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00808-x)

[Tejaas](https://github.com/soedinglab/tejaas): A reverse regression model to detect trans-eQTLs. [(read more here)](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02361-8)

Find the complete list of our open-source software in our the [GitHub](https://github.com/soedinglab)

**Webservers:**

Some of our tools are available as webservice in [MPI Bioinformatics Toolkit](https://toolkit.tuebingen.mpg.de/) of our colleagues from Tübingen.

MMSeqs2 server for fast sequence similarity search: <https://search.mmseqs.com/search>

Datenbanken fuer MMseqs2 software: <https:/data.mmseqs.com> [https://data.mmseqs.com/ "mmseqs2 databases"](https://www.mpibpc.mpg.de/data.mmseqs.com)

BaMMmotif2 webserver for motif discovery and analysis in DNA and RNA sequences: [https://bammmotif.soedinglab.org](https://bammmotif.soedinglab.org/)

Bipartite motif finder for protein-RNA-binding (BMF): [https://bmf.soedinglab.org](https://bmf.soedinglab.org/)

Data server (for HH-suite etc): [Data Server](http://wwwuser.gwdg.de/~compbiol/data/ "Data-Server")
