---
widget: blank
widget_id: Softwares
headless: true
weight: 50
title: Softwares
active: true
design:
  columns: "2"
  background:
    text_color_light: false
    image_darken: 0
---

**Software:**

[HH-suite for sensitive protein sequence searching using hidden Markov models](https://github.com/soedinglab/hh-suite "hhsuite GitHub") [https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-019-3019-7](paper)

[MMseqs2 ultra fast and sensitive sequence searching and clustering: https://mmseqs.com](https://mmseqs.com/ "mmseqs2")[https://academic.oup.com/bioinformatics/article/32/9/1323/1744460](paper)

[Datenbanken fuer MMseqs2 software: https:/data.mmseqs.com](https://www.mpibpc.mpg.de/data.mmseqs.com)[](https://data.mmseqs.com/ "mmseqs2 databases")

[_De novo_ protein-level assembler for metagenomic samples](https://github.com/soedinglab/plass)[https://www.nature.com/articles/s41592-019-0437-4](paper)

[CCMpred: protein residue-residue Contacts from Correlated Mutations](https://github.com/soedinglab/CCMpred "compared") [https://academic.oup.com/bioinformatics/article/30/21/3128/2422261](paper)

[WiSH: Who is the host? prediction of phage-host relationships](https://github.com/soedinglab/wish) [https://academic.oup.com/bioinformatics/article/33/19/3113/3964377](paper)

[kClust: A fast and sensitive clustering method for protein sequences](https://github.com/soedinglab/kClust)[https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-14-248](paper)

[spacepharer: sensitive identification of phages from crispr spacers in prokaryotic hosts](https://github.com/soedinglab/spacepharer) [https://academic.oup.com/bioinformatics/article/37/19/3364/6207963](paper)

[MetaEuk—sensitive, high-throughput gene discovery, and annotation for large-scale eukaryotic metagenomics](https://github.com/soedinglab/metaeuk)[https://microbiomejournal.biomedcentral.com/articles/10.1186/s40168-020-00808-x](paper)

[Tejaas: reverse regression increases power for detecting trans-eQTLs](https://github.com/soedinglab/tejaas)[https://genomebiology.biomedcentral.com/articles/10.1186/s13059-021-02361-8](paper)

Find the complete list of our open-source software in our the [GitHub](https://github.com/soedinglab)

**Webservers:**

Some of our tools are available as webservice in [MPI Bioinformatics Toolkit](https://toolkit.tuebingen.mpg.de/) of our colleagues from Tübingen.

MMSeqs2 server for fast sequence similarity search: <https://search.mmseqs.com/search>

BaMMmotif2 webserver for motif discovery and analysis in DNA and RNA sequences: [https://bammmotif.soedinglab.org](https://bammmotif.soedinglab.org/)

Bipartite motif finder for protein-RNA-binding (BMF): [https://bmf.soedinglab.org](https://bmf.soedinglab.org/)

Data server (for HH-suite etc): [Data Server](http://wwwuser.gwdg.de/~compbiol/data/ "Data-Server")
